pkg/utils/version.go

```diff
@@ -18,23 +21,30 @@ var (
 	BRGitBranch      = "None"
 )
 
-// LogBRInfo prints the BR version information.
+//go:linkname goVersion runtime.buildVersion
+var goVersion string
```

Comment:
why not just use [`runtime.Version()`](https://golang.org/pkg/runtime/#Version)? They are [equivalent](https://github.com/golang/go/blob/cfe3cd903f018dec3cb5997d53b1744df4e53909/src/runtime/proc.go#L14). (perhaps update TiDB too)